test/builtin_dataset_mocks.py

```diff
@@ -1431,3 +1431,48 @@ def stanford_cars(info, root, config):
         make_tar(root, "car_devkit.tgz", devkit, compression="gz")
 
     return num_samples
+
+
+@register_mock
+def sun397(info, root, config):
+    images_root = root / "SUN397"
+    images_root.mkdir()
+
+    categories = ["abbey", "airplane_cabin", "wrestling_ring/indoor"]
```

Comment:
For the category selection it is good practice to select ones with different properties. I've replaced some of the old ones with ones with underscores. Plus, I usually take categories from the start and the end of the list of categories.